components/service/telemetry/src/test/java/org/mozilla/telemetry/measurement/OperatingSystemVersionMeasurementTest.java

```diff
@@ -25,7 +25,7 @@ public void testValue() {
 
         final String version = (String) value;
         assertFalse(TextUtils.isEmpty(version));
-        assertEquals("27", version); // 16 is the default API level of this Robolectric version
+        assertEquals("28", version);
```

Comment:
Maybe we can have a constant for that, this way we have only one point of changing